docs/.vuepress/components/metadata.vue

```diff
@@ -89,6 +89,10 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+    margin-bottom: spacingDouble;
+    padding-bottom: spacingHalf;
+    border-bottom: 1px solid $borderColor;
```

Comment:
weet jij waarom het hier `$borderColor` is en verderop `$accentColor` maar de andere variabelen hebben geen `$`? Heb jij dat bedacht of moet dat zo met Stylus?

Reply:
Variabelen zonder `$` zijn default variabelen die ik zo kan overschrijven. Variabelen met `$` hebben we zelf geïntroduceerd.

Reply:
weer wat geleerrdd